detector/response_cache.go

```diff
@@ -0,0 +1,11 @@
+package detector
+
+import (
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/iam"
+)
+
+type ResponseCache struct {
```

Comment:
__[golint]__
* [golint] exported type ResponseCache should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->